src/services/pcn-iptables/src/defines.h

```diff
@@ -124,7 +124,7 @@ const uint8_t DSTPORT = 4;
 // apply Horus mitigator optimization if there are at least # rules
 // matching the pattern, at ruleset begin
 const uint8_t MIN_RULE_SIZE_FOR_HORUS = 1;
-const uint8_t MAX_RULE_SIZE_FOR_HORUS = -1;  // not used
+const uint8_t MAX_RULE_SIZE_FOR_HORUS = 2048;  // not used
```

Comment:
Is this still not used?

Reply:
Forgot to remove this comment.

Reply:
I just got a compilation warning here, 2048 is too big for a `uint8_t`, I really don't know how this is working.

Reply:
@sebymiano Isn't something you already fixed on some of your branch?

Reply:
I just changed it to `uint32_t` to remove the compilation warning but I haven't tested it.
I guess it should be ok to change.

Reply:
@mauriciovasquezbernal @frisso @sebymiano It was my mistake during code alignment (the code is correct in the firewall, not in iptables).
I've opened a PR for that #163